app/page.tsx

```diff
@@ -21,9 +21,9 @@ export default function Home() {
             <hr className="my-12 mx-auto max-w-7xl  h-px border-t-0 bg-transparent bg-gradient-to-r from-transparent via-neutral-500 to-transparent opacity-25 dark:via-neutral-400" />
 
             <Mission />
-            <PrimaryFeatures />
-            <WhoAreWe />
+            {/* <PrimaryFeatures /> */}
```

Comment:
Its good practise to leave a comment as to why this is temporarily commented out

Reply:
We will hide this section from the landing page and reintroduce it later once 'Bitcoin Topics' is released